visallo/visallo
# fix newly added dynamic relationship not setting parent to root ontology parent if parent is not specified

Testing Instructions: 
- Make sure you can add comments to a newly created edge that you don't specify a parent for. This will make sure we are adding the root ontology parent correctly.

Points of Regression: N/A

NO CHANGELOG
